### PR TITLE
User Profile Mobile Club Cards

### PR DESCRIPTION
**Pull request by sigmachirality**
This PR also fixes the bug where clicking "Manage" on a club in the clubs tab of the user profile doesn't do anything.

Currently this looks like:
Screenshot from 2020-01-18 23-11-30

### CODE REVIEW

**Comment by ezwang** [comment]
Screenshot of desktop view:

image

File: frontend/components/Settings/ClubTabTable.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
+import { DARK_GRAY } from '../../constants/colors'
+import { BODY_FONT } from '../../constants/styles'
+import s from 'styled-components'
+import Toggle from './Toggle'
+import { Icon } from '../common'
+import ReactTooltip from 'react-tooltip'
+import Link from 'next/link'
```
Comment (cacabo) [inline frontend/components/Settings/ClubTabTable.js]: Put package imports before relative ones

File: frontend/components/Settings/ClubTab.js [hunk 2]
```diff
@@ -1,120 +1,40 @@
...
+import ClubTabTable from './ClubTabTable'
+import ClubTabCards from './ClubTabCards'
 
-const Table = s.table`
-  font-family: ${BODY_FONT};
-  font-size: 16px;
-  overflow: scroll;
-  color: ${DARK_GRAY} !important;
+const ClubTable = s(ClubTabTable)`
+  @media (max-width: 768px) {
```
[the remark this answers is not in the document]
Comment (sigmachirality) [reply frontend/components/Settings/ClubTab.js]: Switched to SM!

File: frontend/components/Settings/ClubTabCards.js [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+  background-color: ${CLUBS_BLUE};
+  color: ${SNOW};
+  justify-content: flex-start;
+`
+
+const LeaveButton = s.a`
+  border: none;
+  border-radius: ${BORDER_RADIUS};
+  background-color: ${CLUBS_RED};
+  color: ${SNOW};
```
[the remark this answers is not in the document]
Comment (sigmachirality) [reply frontend/components/Settings/ClubTabCards.js]: There's no Button component currently, there's actually a story in clubhouse to make one!

**Pull request merged**